traitsui/editors/tuple_editor.py

```diff
@@ -73,9 +58,10 @@ class ToolkitEditorFactory(EditorFactory):
     # 'enter_set' metadata or an editor defined.
     enter_set = Bool(False)
 
-#-------------------------------------------------------------------------
-#  'SimpleEditor' class:
-#-------------------------------------------------------------------------
+    # The validation function to use for the Tuple. If the edited trait offers
+    # already a validation function then the value of this trait will be
+    # ignored.
+    fvalidate = Callable
```

Comment:
It has been a while... So initially I had it a validator but then I changed it to fvalidate to be similar to the Property fget, fset, fvalidate